src/quickfix.c

```diff
@@ -3261,8 +3261,35 @@ qf_jump_edit_buffer(
 		prev_winid == curwin->w_id ? curwin : NULL);
     }
     else
+    {
+	if (!forceit && curwin->w_p_wfb)
+	{
+	    if (qi->qfl_type == QFLT_LOCATION)
+	    {
+	        // Location lists cannot split or reassign their window
+	        // so 'winfixbuf' windows must fail
+	        semsg("%s", e_winfixbuf_cannot_go_to_buffer);
+	        return QF_ABORT;
```

Comment:
and here please

Reply:
This block is used in these tests

Test_lNext()
Test_lNfile()
Test_ll()
Test_llast()
Test_lnext()
Test_lnfile()
Test_lpfile()
Test_lprevious()
Test_lrewind()